docs/quantum_research/solver_parameters.rst

```diff
@@ -1002,7 +917,7 @@ composite instead.
 programming_thermalization
 ==========================
 
-Sets the time, in microseconds |anneal_time_parameter_granularity|, to wait
+Sets the time, in microseconds with a resolution of 0.01 :math:`\mu s`, to wait
```

Comment:
Let's keep this replacement so if in the future our resolution increases, we can change that in one place and not have to search through text to change in multiple places